.codecov.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+codecov:
+  notify:
+    after_n_builds: 1  # Number of test matrix+lint jobs uploading coverage
+    wait_for_ci: false
+
+  require_ci_to_pass: false
+
+  token: 2b8c7a7a-7293-4a00-bf02-19bd55a1389b  # repo-scoped
```

Comment:
please add comment around why is this token in plain text and the scope and permission of this token